content/docs/1.4.2/deploy/upgrade/upgrade-engine.md

```diff
@@ -28,15 +28,15 @@ The `iSCSI` frontend does not support live upgrades.
 Live upgrade should only be done with healthy volumes.
 
 1. Select the volume you want to upgrade.
-2. Click `Upgrade Engine` in the drop down.
+2. Click `Upgrade Engine` in the drop-down.
 3. Select the engine image you want to upgrade to.
-    1. Normally it's the only engine image in the list, since the UI exclude the current image from the list.
+    1. Normally it's the only engine image in the list, since the UI excludes the current image from the list.
 4. Click OK.
 
-During the live upgrade, the user will see double number of the replicas temporarily. After upgrade complete, the user should see the same number of the replicas as before, and the `Engine Image` field of the volume should be updated.
+During the live upgrade, the user will see double the number of replicas temporarily. After the upgrade is complete, the user should see the same number of replicas as before, and the `Engine Image` field of the volume should be updated.
 
-Notice after the live upgrade, Rancher or Kubernetes would still show the old version of image for the engine, and new version for the replicas. It's expected. The upgrade is success if you see the new version of image listed as the volume image in the Volume Detail page.
+Notice after the live upgrade, Rancher or Kubernetes would still show the old version of image for the engine, and new version for the replicas. It's expected. The upgrade is successful if you see the new version of image listed as the volume image in the Volume Detail page. In order to clear out the old instance manager engine pod for a volume that was upgraded, the volume needs to be detached and reattached by scaling the workload down and then scaling it back up.
```

Comment:
This was the explanation I received from @PhanLe1010 ...
If we look closer, there are 2 different components: the instance-manager pod (with image longhornio/instance-manager:xxx and the volume's engine (with image longhornio/longhorn-engine:xxx)
When user upgrade engine for volume, the second component (volume's engine) changes. But the first one will be the same because the new engines are still living inside the old instance manager pods (this is the design that makes live engine upgrade possible)
In order for the old instance-manager-e pod to be removed, the volume need to be detached and reattached. This means the workload needs to be scale down and up (edite

Reply:
I completely agree with @PhanLe1010's explanation. I'd only like us to clarify what the user can look at with "Rancher or Kubernetes". The only old thing they'll see after a live upgrade is an old instance-manager. The actual engine object itself will show the new version, as will the replica objects and the replica instance-manager. My suggested change just adds a bit of clarifying context.